frontend/src/stores/DataStore.js

```diff
@@ -284,6 +284,12 @@ class DataStore {
         this.datasets[index] = dataset;
         this.toggleDatasetModal();
     }
+
+    @computed get getSelectedDataTabularForm() {
```

Comment:
this doesn't work as implemented; column order is unstable, so sometimes N will be the first column, and sometimes dose. I will fix.